routes/api/contacts.js

```diff
@@ -0,0 +1,124 @@
+const express = require('express');
+const {
+  listContacts,
+  getContactById,
+  removeContact,
+  addContact,
+  updateContact,
+  updateStatusContact,
+} = require('../../service/contactsService');
+
+const validate = require('../../validation/validation');
+
+const router = express.Router();
+
+router.get('/', async (req, res, next) => {
+  try {
+    const ownerId = req.user._id;
+    const contacts = await listContacts(ownerId);
+    res.status(200).json({ contacts });
+  } catch (error) {
+    next(error);
+  }
+});
+
+router.get('/:contactId', async (req, res, next) => {
+  try {
+    const { contactId } = req.params;
+    const ownerId = req.user._id;
+    const contact = await getContactById(contactId, ownerId);
+    if (contact) {
+      return res.status(200).json({ contact });
+    } else {
+      return res
+        .status(404)
+        .json({ message: `Contact with ID ${contactId} not found` });
+    }
```

Comment:
Jeśli robisz return tak jak tutaj to możesz kod trochę bardziej uprościć, np: 
```suggestion
    if (contact)  return res.status(200).json({ contact });
   
      return res
        .status(404)
        .json({ message: `Contact with ID ${contactId} not found` });
```

Reply:
Ten powyższy kod robi dokładnie to samo a nie ma niepotrzebnych zagnieżdżeń, łatwiej się go czyta. Jeśli kontakt będzie to przerwiesz w linijce 30 a jeśli nie będzie to w 32.